From 7cf2b84ed275ae4dc76d8cf7853cc499a72cff39 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Wed, 20 Feb 2019 01:34:55 +0530
Subject: [PATCH 7/8] Allow using public API when building with MSVC

* pthread.h is not actually used anywhere in the public API

* stdint.h and inttypes.h are provided by MSVC now

* The define is _WIN32 not WIN32
---
 common/filelist_win32.maf | 2 --
 srtcore/logging_api.h     | 1 -
 srtcore/platform_sys.h    | 2 +-
 3 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/common/filelist_win32.maf b/common/filelist_win32.maf
index e6864de..2762c65 100644
--- a/common/filelist_win32.maf
+++ b/common/filelist_win32.maf
@@ -4,8 +4,6 @@ win/syslog_defs.h
 #
 # These are included by platform_sys.h header contained in ../srtcore/filelist.maf
 #
-win/inttypes.h
-win/stdint.h
 win/unistd.h
 
 SOURCES
diff --git a/srtcore/logging_api.h b/srtcore/logging_api.h
index 74ae792..82c6316 100644
--- a/srtcore/logging_api.h
+++ b/srtcore/logging_api.h
@@ -24,7 +24,6 @@ written by
 #include <iostream>
 #endif
 
-#include <pthread.h>
 #ifdef WIN32
 #include "win/syslog_defs.h"
 #else
diff --git a/srtcore/platform_sys.h b/srtcore/platform_sys.h
index ce546d2..b5cd910 100644
--- a/srtcore/platform_sys.h
+++ b/srtcore/platform_sys.h
@@ -10,7 +10,7 @@
 #ifndef INC__PLATFORM_SYS_H
 #define INC__PLATFORM_SYS_H
 
-#ifdef WIN32
+#ifdef _WIN32
    #include <winsock2.h>
    #include <ws2tcpip.h>
    #include <ws2ipdef.h>
-- 
2.23.0

